tests/flows/conftest.py

```diff
@@ -121,22 +121,32 @@ def vespa_app(
     print("\nSetting up Vespa connection...")
     app = Vespa(mock_vespa_credentials["VESPA_INSTANCE_URL"])
 
-    subprocess.run(
```

Comment:
Reverting the changes to this file and `tests/flows/fixtures/CCLW.executive.4934.1571.json` was helpful for me during debugging the test to isolate the issue. We can add back in if required.

Reply:
Will do! Thanks for the explanation.